doc/Py3K.rst.txt

```diff
@@ -394,28 +394,6 @@ There are a couple of places that need further attention:
   In some cases, this returns a buffer object on Python 2. On Python 3,
   there is no stand-alone buffer object, so we return a byte array instead.
 
-The Py2/Py3 compatible PyBufferMethods definition looks like::
```

Comment:
We decided not to touch this document, right @charris?

Reply:
Right. Maybe we should have a place to put "history". The original intent of the document was to aid others in making the transition at the C level. NumPy was one of the first to support Python 3.

Reply:
`doc/Py3k.rst.txt` is not linked into the html documentation as far as I can tell. So we could move it to an `archive` directory

Reply:
reverted.